src/Mvc/Mvc.ViewFeatures/src/Infrastructure/DefaultTempDataSerializer.cs

```diff
@@ -85,7 +85,7 @@ public override byte[] Serialize(IDictionary<string, object> values)
 
             using (var bufferWriter = new ArrayBufferWriter<byte>())
             {
-                var writer = new Utf8JsonWriter(bufferWriter);
+                using var writer = new Utf8JsonWriter(bufferWriter);
```

Comment:
@pranavkm FYI

Reply:
Look at you with your swanky new C# 8 features.

Reply:
You know @BrennanConroy's only excited because it's basically a C++ feature (RAII) 😉